src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,4 @@
+package mate.academy.model;
+
+public class MivieSession {
```

Comment:
There is a typo in the class name. It should be `MovieSession` instead of `MivieSession`.